src/smc-webapp/projects.cjsx

```diff
@@ -525,7 +525,8 @@ class ProjectsStore extends Store
             return 0
         others = []
         for i in v
-            if not i.deleted and (show_hidden or not i.get('users').get(account_id).get('hide'))
+            # Deleted projects have a map node " 'deleted': true ". Standard projects do not have this property.
+            if (not i.get('deleted')) and (show_hidden or not i.get('users').get(account_id).get('hide'))
```

Comment:
Yes. The variable 'i' is a map element in a list and since 'i.deleted' didn't exist 'not i..deleted' was always true. The deleted property was working in other parts of the site so I figured it probably was not being called properly here.